Resources/Private/Partials/PageView.html

```diff
@@ -67,11 +67,16 @@ <h2>
                                         <f:if condition="{isNewspaper} == 'newspaper_issue'">
                                             <!-- it's a newspaper! -->
                                         </f:if>
-                                        <f:cObject typoscriptObjectPath="plugin.tx_dlf_pageview" />
-                                        <f:cObject typoscriptObjectPath="plugin.tx_dlf_audioplayer" />
-                                        <div class="fulltext-container">
-                                            <div id="tx-dlf-fulltextselection"></div>
```

Comment:
Why fulltextselection is removed?

Reply:
The fulltext section is always displayed when the page view is rendered (https://github.com/slub/dfg-viewer/blob/2199c0cb2958f65feaa1b19e00648347ee07bae8/Resources/Private/Plugins/Kitodo/Templates/PageView/Main.html), so I moved it there to make the partial a bit cleaner. Don’t you think that’s a good idea?